src/util.cpp

```diff
@@ -1866,7 +1866,7 @@ QString GetVersionAndNameStr ( const bool bWithHtml )
     }
     else
     {
-        strVersionText += " *** ";
+        strVersionText += "\n *** ";
```

Comment:
This is what I _currently_ see:
![image](https://user-images.githubusercontent.com/1549463/147381292-a90107f9-2c3f-4300-b87f-1490b8330a65.png)
...
![image](https://user-images.githubusercontent.com/1549463/147381313-b414993c-4c3e-48c1-9733-51dc4447e782.png)
... and no command prompt at the end.

Do you understand _why_ this happens?

It's because of how Windows launches a Qt process.  The process detaches to run in the background, letting `cmd` print the next prompt - and it does that before Jamulus can print the help text (usually -- it _could_ appear anywhere during the flow).

Adding a newline here won't allieviate the problem, in my view, so I'd rather it wasn't changed.

Reply:
The -h has been fixed (add \n at the start) in a previous PR. This was not done for -v (yet).

This is what you see when using -v:
![command-v-output](https://user-images.githubusercontent.com/13550012/147383833-a3c3edc7-3382-4715-b011-a539ebbb4bac.PNG)

I understand why there is no command prompt at the end. That is how Qt and Windows work when using GUI application. If it was running as a console application, the command prompt would appear.

Apology for the confusing, but I did not want to re-introduce what was fixed/added in a previous PR #1759 for the help text.

Reply:
Sorry, yes, I misread your description.

Still, I'd rather the Windows-specific behaviour were made clear by having it in a `#ifdef`.

Reply:
Agree, really specific to windows here. So can add the ifdef. The popup GUI would be very nice, as that is how most windows apps do it these days.